interface-definitions/https.xml.in

```diff
@@ -60,6 +60,32 @@
                   <multi/>
                 </properties>
               </leafNode>
+              <leafNode name="allow-client">
```

Comment:
I wonder if there should be “node” before, I.e. “allow-client address x.x.x.x” vs “allow-client x.x.x.x”

as we already have “allow-client address x.x.x.x” for other commands we can add XML and reuse them for new XML via “include”

@dmbaturin @c-po your thoughts?

Reply:
Good point, @sever-sever; I'll fix.

Reply:
Fixed; thanks @sever-sever